### DISCUSSION TITLE
Screen to PNG?

### DISCUSSION THREAD

**Discussion by kevinfields777** [NickeManarin/ScreenToGif#1290] [Ideas]
Have you thought about an option to output to animated PNG instead of animated GIF?

**Comment by pawlos** [comment]
@kevinfields777 hmm, you mean this?
![image](https://github.com/NickeManarin/ScreenToGif/assets/1296768/b76c532e-930d-4db6-958b-5293559f2f2e)

**Reply by kevinfields777** [reply]
AHA! I'm blind, the Save As page was scrolled all the way to the bottom and I didn't realize there were more options.  Thank you!
